wallets/cosmostation-extension/src/extension/client.ts

```diff
@@ -32,6 +33,16 @@ export class CosmostationClient implements WalletClient {
     return this.client.providers.keplr;
   }
 
+  async suggestToken({ chainName, tokens }: SuggestToken) {
```

Comment:
can we rename this to `suggestCW20Token`, or, `suggestToken` should take a parameter that is an enum or something, like `type: 'cw20' | 'erc20'`, etc.

Reply:
Yes, that's a good idea, I'll add another parameter